dashboard/deploy_dashboard_gcloud

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash -e
 
-if [ $# != 1 ]; then
-    echo Usage: $0 [project_id]
+if (( $# < 1 )); then
+    echo Usage: $0 PROJECT_ID [options]
```

Comment:
What options are potentially interesting here?  I've never had to add anything else, so just wondering... I'm a bit worried about making this too flexible since ideally the cloud-deploy part would be very regular/consistent...

Reply:
I looked into it, I don't think any. I made some assumptions based on the names of some things but I don't think they're relevant anymore, so it's only the service account then.